templates/zerver/help/browse-and-join-streams.md

```diff
@@ -1,28 +1,29 @@
 # Browse and subscribe to streams
 
 On Zulip, users communicate with each other by sending messages to
-conversation streams, which are similar to conversation threads.  New
+streams, which are similar to conversation threads.  New
 users are automatically subscribed to the organization's default
 streams; however, users may want to subscribe to additional streams or
 browse the list of streams that they subscribe to.
 
 ## Browse streams
 
 1. Click the cog (<i class="icon-vector-cog"></i>) icon next to **Streams**
-in the left sidebar. A tooltip labeled **Subscribe, add, or configure streams**
-will appear upon hovering above the cog.
+in the left sidebar. A tooltip labeled **Subscribe, add, or
+configure streams** will appear upon hovering above the cog.
 
     ![Streams cog and tooltip](/static/images/help/streams-1.png)
 
 2. After clicking the cog (<i class="icon-vector-cog"></i>) icon, the
-[Subscriptions](/#subscriptions) page will now appear, showing all streams that
-you've subscribed to by default.
+[Subscriptions](/#subscriptions) page will now appear, showing all streams
+that you've subscribed to by default.
 
-    ![[Subscriptions page](/static/images/help/streams-overview.png)
+    ![Subscriptions page](/static/images/help/streams-overview.png)
 
-Here, you can explore all the public streams in the organization. You can also
-[create a new stream](create-a-stream) by clicking the plus
-(<i class="icon-vector-plus"></i>) icon to the right of the **Filter Streams** input.
+    Here, you can explore all the public streams in the organization.
+You can also [create a new stream](create-a-stream) by clicking the plus
+(<i class="icon-vector-plus"></i>) icon to the right of the
```

Comment:
>  icon to the right of the

Keep in mind that the UI may change — don’t describe it in more detail than is needed.